noty-api/application/build.gradle

```diff
@@ -15,6 +15,7 @@ repositories {
 dependencies {
     // Data module
     implementation project(":data")
+    implementation "com.zaxxer:HikariCP:$hikariVersion"
```

Comment:
Why include this in the application module? I guess it's only needed in the data module.

Reply:
Was using HikariConfig in Application module but it was not necessary so removed it.

Reply:
Was using HikariConfig in Application module but it was not necessary so removed it.